test/recipes/80-test_cms.t

```diff
@@ -800,6 +800,7 @@ subtest "CAdES ko tests\n" => sub {
       SKIP: {
         my $skip_reason = check_availability($$_[0]);
         skip $skip_reason, 1 if $skip_reason;
+        1 while unlink "cades-ko.txt";
```

Comment:
It's required on VMS, where files may exist in multiple generations (so `cades-ko.txt;1`, `cades-ko.txt;2`, ...)

Reply:
Regarding deleting all but for the final loop, you're right about that.  However, the array we use has only one element.  The function `runner_loop` handles that better, but isn't quite suitable for this particular test because of the second negative test.  I've thoughts on reworking `runner_loop`, but that's overkill for this fix.